modules/python/init.zsh

```diff
@@ -65,6 +110,12 @@ if (( $+commands[pip] )); then
   source "$cache_file"
 
   unset cache_file
+
+  # Load auto workon cwd hook
+  if zstyle -t ':prezto:module:python' enable-virtualenv-auto-workon-cwd 'yes'; then
```

Comment:
I've been using [autoenv](https://github.com/kennethreitz/autoenv) to accomplish roughly the same thing in the past, and that repo has roughly half the stars that Prezto itself has, so it's not a tiny or unknown project.  So barring other reservations, I'd go for ever shorter like, just `:prezto:module:python autoenv`.  My only problem with that one is that almost sounds like it's activating Reitz' `autoenv` tool.  Is that a valid concern?

Personally, I at least don't feel like the `cwd` really needs to be part of the name.  If you didn't want `autoenv`, my next vote would be for `auto-virtualenv`.  iunno, just my $0.02.